schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              },
+              "administrator_comments": {
+                "title": "Administrator comments",
+                "description": "Comments from by arXiv administrators",
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/comment"
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              },
+              "legacy_comments": {
+                "description": "Unstructured comments from authors and/or arXiv administrators.",
+                "$ref": "#/definitions/comment"
+              }
+            }
+          },
+          "journal_ref": {
+            "title": "Journal reference",
+            "type": "string",
+            "minLength": 1
+          },
+          "publisher_dois": {
+            "title": "Publisher DOIs",
+            "description": "The journal DOI(s).",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/doi"
+            },
+            "minItems": 1
+          },
+          "arxiv_doi": {
+            "title": "arXiv DOI",
+            "description": "The arXiv-issued DOI.",
+            "$ref": "#/definitions/doi"
+          },
+          "report_num": {
+            "title": "Report number",
+            "type": "string",
+            "minLength": 1
+          },
+          "msc_classes": {
+            "title": "List of MSC classes",
+            "description": "Classifications from American Mathematical Society Mathematical Subject Classification (MSC).",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "Primary 14N35, Secondary 14H10, 14H20, 14J26",
+              "Primary 46F30. Secondary: 44A10, 46S10, 46F10, 12J15, 12J25, 16W60",
+              "46B20, 47H10, 54B20, 54F15, 68U05"
+            ],
+            "$comment": "TODO: consider array of an enumerated definition instead. Legacy input is inconsistent."
+
+          },
+          "acm_classes": {
+            "title": "List of ACM classes",
+            "description": "Classifications from ACM Computing Classification System.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "C.2.0; C.2.2; C.2.3; C.2.6",
+              "I.3.5",
+              "G.1.6; F.1.1; I.2.6; G.1.3"
+            ],
+            "$comment": "TODO: consider array of an enumerated definition instead. Legacy input is inconsistent."
+          },
+          "author_links": {
+            "title": "List of author links",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/author_link"
+            },
+            "minItems": 1,
+            "uniqueItems": true,
+            "$comment": "should this be version-specific?"
+          },
+          "external_publication_status": {
```

Comment:
external_publication_status: This is not data that belongs to arXiv. 

How do we see this getting entered and maintained? Seems like we need an automated system where the data is fed from publishers. Like a author puts the arXiv ID into the journal submission form, then the journal calls https://arxiv.org/abs/1234.12345/update_publish_status with an auth token.

Reply:
This field is intended to bring structure to information that is already routinely being placed by authors in the catch-all `comments` field. This is a heavily requested update.